### PR TITLE
[NEW] monespacesante.fr

### PR DESCRIPTION
**Pull request by irew**
-   **Domain Name**: 
monespacesante.fr
-   **Purpose**:
Health
-   *
View recent photos
*Relevance**:

-   **Additional Information**:
Only on mobile

### CODE REVIEW

File: resources/compatible-domains.json [hunk 1]
```diff
@@ -64,5 +64,6 @@
     "payfit.com",
     "tiktok.com",
     "yahoo.co.jp",
-    "yahoo.com"
+    "yahoo.com",
+    "monespacesante.fr"
```
Comment (Mikescops) [inline resources/compatible-domains.json:68]: can you order it please :)

Comment (irew) [reply resources/compatible-domains.json:68]: The list is not ordered at all

Comment (irew) [reply resources/compatible-domains.json:68]: Want me to order the entire list in the same PR?

Comment (Mikescops) [reply resources/compatible-domains.json:68]: oh damn, i thought i ordered it at some point, yes please do then 👍

Comment (irew) [reply resources/compatible-domains.json]: done

**Pull request merged**